contracts/IReputationMiningCycle.sol

```diff
@@ -0,0 +1,144 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity ^0.4.23;
+pragma experimental "v0.5.0";
+
+
+contract IReputationMiningCycle {
```

Comment:
Some gas golfing can be done with this interface to replace the import in `ColonyNetworkStaking` and also help decouple the mining client.

Reply:
Done for the mining client, but not for `ColonyNetworkStaking`.